### ISSUE TITLE
Bug only on safari

### ISSUE THREAD

**Issue by ibrunotome** [LucasLeandro1204/vue-content-loading#17]
Hi, can you help me with this? If I reload the page the loading come from this: 
<img width="411" alt="screenshot 2018-08-02 13 52 34" src="https://user-images.githubusercontent.com/4256471/43598685-aae6971e-965b-11e8-9a32-23374360f9f1.png">

to this:
<img width="479" alt="screenshot 2018-08-02 13 52 21" src="https://user-images.githubusercontent.com/4256471/43598699-b353bef4-965b-11e8-89fc-df2d89eff0b9.png">

** This happening only on safari and only if I refresh the page (cmd + r "F5"), no errors or warnings on console.

**Comment by lourdrivera123** [comment]
Are you still experiencing this issue?

**Comment by LucasLeandro1204** [comment]
I couldn't reproduce this issue =/

**Comment by ibrunotome** [comment]
Yes, still happening, I'm using nuxt and this happen even after nuxt build for production. This also happen with this package: https://github.com/egoist/vue-content-loader.

So I don't know if i can make something to solve this.

**Comment by slavensaka** [comment]
Experiencing the same issue on with nuxt. Going from page to page it works perfect, but when reload that page with content-load it's turns fully black.
on the parent div i have <div v-if="showProgress"  class="spacesProgress"> ... content load </div>
Removing the v-if or v-show works correctly. Issue is only on Edge and Safari. 


![capture](https://user-images.githubusercontent.com/8411116/52564828-0677c880-2dfd-11e9-95f4-a50e51de82ff.PNG)

**Comment by jetaix** [comment]
@LucasLeandro1204 any news about this issue ?

**Comment by victorybiz** [comment]
Has anyone solved this bug on safari?
